scripts/content.js

```diff
@@ -139,13 +139,13 @@ const observer = new MutationObserver((mutations, instance) => {
   fix();
 
   if (!isPreview) {
-    mutations.forEach((mutation) => {
-      mutation.addedNodes.forEach((node) => {
-        const valid = check(node);
-
-        if (valid && node.matches(selectors)) node.outerHTML = "";
-      });
-    });
+    const nodes = mutations.map((mutation) => [...mutation.addedNodes])
```

Comment:
I really like the "more functional" approach but it seems a less performant way to iterate these arrays. Previously we had only two iterations using two `for each` loops and now we have four: `map`, `flat`, `filter` and finally a `for of` loop.

Reply:
Oh, I clearly see what you mean. So sorry. I'll revert those changes